### DISCUSSION TITLE
Input as part of node

### DISCUSSION THREAD

**Discussion by sklinov** [antvis/G6#3558] [Q&A]
Does anyone know of a way of having something like HTML <input> element as a part of a custom node in `canvas` rendering mode?

**Comment by sklinov** [comment]
Well, answering my own question:
- docs don't have any mentions of that's directly possible
- here's what I came up with (super-simple POC) - that for sure can be elaborated into a more input-like looking element 
https://codesandbox.io/s/g6-input-4x95u9

**Comment by Yanyan-Wang** [comment]
Maybe just append an input DOM element upon the canvas. About mounting DOM onto canvas:  https://g6.antv.vision/en/docs/manual/advanced/coordinate-system#using-the-three-coordinate-systems
